DeAceroDLabs/marketplace-frontend
# Feature/dmp 412

## What is done

[[DMP-412] Navigation Footer UI](https://deacero.atlassian.net/browse/DMP-412)

## How is done

- Created a `Footer` component that is available across the app.
- Inside the footer add a section for contact support, webiste related info and frequently asked questions.

## How to test

1. Run the project via `npm start`
2. Login.
3. You should see the Home Screen with the footer.
